code/modules/mob_spawn/corpses/mining_corpses.dm

```diff
@@ -137,6 +137,7 @@
 /obj/effect/mob_spawn/corpse/human/legioninfested/skeleton/charred/special(mob/living/carbon/human/spawned_human)
 	. = ..()
 	spawned_human.color = "#454545"
+	spawned_human.AddComponent(/datum/component/storm_hating)
```

Comment:
You're gonna wanna have the component check for `mob_login` -> `qdel(src)` otherwise the small group of people brain swapping into these are gonna be so owned